test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -2,6 +2,8 @@ Could not find tests for function assemblyproperty
 Could not find tests for function pltsql_call_handler
 Could not find tests for function pltsql_inline_handler
 Could not find tests for function pltsql_validator
+Could not find tests for function sys.babelfish_fts_contains_rewrite
```

Comment:
Why do we have to add those into expected_create ?
Can we add related test cases ?

Reply:
Since this function is not directly used my customer, we are not writing test cases for this function, rather we've added test cases for the underlying fulltext search feature.